doc/swagger.v1.yml

```diff
@@ -285,6 +310,51 @@ definitions:
         format: date-time
         description: when the transaction was entered
     required: ['id', 'user_id', 'user_name', 'amount', 'note', 'created_at']
+    
+  FinancialSummary:
+    type: object
+    properties:
+      id:
+        type: integer
+      user_id:
+        type: 'integer'
+        default: null
+        description: id of user who entered the transaction (may be <tt>null</tt> for deleted users or 0 for a system user)
+      user_name:
+        type: 'string'
+        default: null
+        description: name of user who entered the transaction (may be <tt>null</tt> or empty string for deleted users or system users)
+      oder_id:
+        type: 'integer'
+        default: null
+        description: id of order group, user is assigned to
+      credit:
+        type: number
+        default: 0
+        description: actual available credit of user/ordergroup
+      balance:
+        type: 'integer'
+        default: 0
+        description: credit + open claim
+      classes:
```

Comment:
let's use the full name, I think that's clearer, so `financial_transaction_classes` (if I understand correctly)